command/init.go

```diff
@@ -856,6 +856,9 @@ func (c *InitCommand) backendConfigOverrideBody(flags rawFlags, schema *configsc
 			// The value is interpreted as a filename.
 			newBody, fileDiags := c.loadHCLFile(item.Value)
 			diags = diags.Append(fileDiags)
+			if fileDiags.HasErrors() {
+				break
```

Comment:
```suggestion
				continue
```

We should `continue` here, instead of `break`, as its possible to pass multiple items and we'd want to catch errors for each.